package.json

```diff
@@ -85,6 +86,8 @@
     "@radix-ui/react-icons": "^1.3.0",
     "@radix-ui/react-label": "^2.1.0",
     "@radix-ui/react-toast": "^1.2.1",
+    "@radix-ui/react-toggle-group": "^1.1.2",
+    "@radix-ui/react-tooltip": "^1.1.8",
```

Comment:
I cannot use `@radix-ui/theme` in the browser extension. We are using a shadow DOM to isolate our styling from the recorded page, but the `<Theme />` component mounts its styles in `document.head`. This means that the theme styles are applied to the recorded page, and not to our shadow DOM. As far as I can tell, there's no way of specifying where the styles should be mounted.